### PR TITLE
Remove terminal specific command from e2e service test file

### PR DESCRIPTION
**Pull request by amitkrout**
## What is the purpose of this change? What does it change?

Update the test file to run it on all oc supported platform including windows 

## Was the change discussed in an issue?
fixes - Part of #1129

## How to test changes?

make test-service-e2e

### CODE REVIEW

File: tests/e2e/utils.go [hunk 1]
```diff
@@ -90,3 +97,21 @@ func getActiveApplication() string {
 	odoActiveApp := strings.Split(reActiveApp.FindString(stdOut), "*")[1]
 	return strings.TrimSpace(odoActiveApp)
 }
+
+func retryingForOutputMatchStringOfHTTPResponse(url, match string, retry, sleep int) bool {
+	for i := 0; i < retry; i++ {
```
Comment (syamgk) [inline tests/e2e/utils.go]: If variable `i` is not being used
use simple for loop without variable, like this
```
for {
}
```

File: tests/e2e/utils.go [hunk 2]
```diff
@@ -90,3 +97,21 @@ func getActiveApplication() string {
 	odoActiveApp := strings.Split(reActiveApp.FindString(stdOut), "*")[1]
 	return strings.TrimSpace(odoActiveApp)
 }
+
+func retryingForOutputMatchStringOfHTTPResponse(url, match string, retry, sleep int) bool {
```
Comment (kadel) [inline tests/e2e/utils.go]: Isn't this the same function as in https://github.com/redhat-developer/odo/pull/1401/files#diff-3ee295477bcffbfddd29e7f409916eb2R101 ?

Comment (amitkrout) [reply tests/e2e/utils.go]: Yes

File: tests/e2e/utils.go (hunk 1 above)
Comment (mohammedzee1000) [reply tests/e2e/utils.go]: It is being used in the loop itself, as the counter.

**Comment by mohammedzee1000** [comment]
/meow
:)

**Comment by stevekuznetsov** [comment]
@mohammedzee1000 since you have approving reviews configured as required in GitHub, our merge system tries to merge but cannot:

```json
{
  "base-sha": "f80a5c4310677690385df9fe232f256c3b2f32c6",
  "branch": "master",
  "component": "tide",
  "controller": "sync",
  "error": "1 review requesting changes and 2 approving reviews by reviewers with write access.",
  "level": "error",
  "merge-targets": [
    1408
  ],
  "msg": "Merge failed: PR is unmergable. Do the Tide merge requirements match the GitHub settings for the repo?",
  "org": "redhat-developer",
  "pr": 1408,
  "repo": "odo",
  "sha": "7f8214e2626de825f6579647090b4f93dd36fc76",
  "time": "2019-03-06T05:45:36Z"
}
```

You'll want to remove those requirements or always ensure they are met before LGTM+approve

**Comment by mohammedzee1000** [comment]
> @mohammedzee1000 since you have approving reviews configured as required in GitHub, our merge system tries to merge but cannot:
> 
> ```json
> {
>   "base-sha": "f80a5c4310677690385df9fe232f256c3b2f32c6",
>   "branch": "master",
>   "component": "tide",
>   "controller": "sync",
>   "error": "1 review requesting changes and 2 approving reviews by reviewers with write access.",
>   "level": "error",
>   "merge-targets": [
>     1408
>   ],
>   "msg": "Merge failed: PR is unmergable. Do the Tide merge requirements match the GitHub settings for the repo?",
>   "org": "redhat-developer",
>   "pr": 1408,
>   "repo": "odo",
>   "sha": "7f8214e2626de825f6579647090b4f93dd36fc76",
>   "time": "2019-03-06T05:45:36Z"
> }
> ```
> 
> You'll want to remove those requirements or always ensure they are met before LGTM+approve

Thanks will do

**Pull request merged**